github_rules/github_branch_policy_override.yml

```diff
@@ -0,0 +1,27 @@
+AnalysisType: rule
+Filename: github_branch_policy_override.py
+RuleID: GitHub.Branch.PolicyOverride
+DisplayName: GitHub Branch Protection Policy Override
+Enabled: false
+LogTypes:
+  - GitHub.Audit
+Tags:
+  - GitHub
+Severity: High
+Description: Bypassing branch protection controls could indicate malicious use of admin credentials in an attempt to hide activity.
+Runbook: Verify that the GitHub admin performed this activity and validate its use.
+SummaryAttributes:
+  - p_any_ip_addresses
+Tests:
+  -
+    Name: GitHub - Branch Protection Policy Override
+    ExpectedResult: true
+    Log:
+      {
+        "actor": "cat",
+        "action": "protected_branch.policy_override",
+        "created_at": 1621305118553,
+        "p_log_type": "GitHub.Audit",
+        "org": "my-org",
+        "repo": "my-org/my-repo"
+      }
```

Comment:
I know it's a pain, but let's try to do one `ExpectedResult: true` and one `ExpectedResult: false` test when possible, which I think it should be pretty straightforward to do here.